iina/AppData.swift

```diff
@@ -93,6 +93,7 @@ struct Constants {
     static let keyBindingInputChanged = Notification.Name("IINAkeyBindingInputChanged")
     static let fileLoaded = Notification.Name("IINAFileLoaded")
     static let historyUpdated = Notification.Name("IINAHistoryUpdated")
+    static let legacyFullScreen = Notification.Name("IINALegacyFullScreen")
```

Comment:
Yep, I was exactly about to do the same thing. Maybe in the next refactor branch.